graphql-server/pkg/queries/proposal.go

```diff
@@ -131,3 +134,30 @@ func (q *ProposalQuery) QueryProposalByIDs(ids []string) ([]*models.Proposal, er
 	}
 	return proposals, nil
 }
+
+func (q *ProposalQuery) QueryProposalDepositTotal(id int, config config.Config) (gql_bigint.BigInt, error) {
+	var res gql_bigint.BigInt
+	rows, err := q.session.DB.QueryContext(q.ctx, fmt.Sprintf(
+		`SELECT
+			sum((deposit.coin).amount::BIGINT)
+		FROM (
+			SELECT
+				unnest(proposal_deposit.amount) AS coin
+			FROM
+				%s.proposal_deposit
+			WHERE
+				proposal_deposit.proposal_id = %d
+			) AS deposit
+		where
+			(deposit.coin).denom = '%s'
+		GROUP BY
+			(deposit.coin).denom;`, config.ChainDatabase.Schema, id, config.Chain.CoinDenom,
```

Comment:
We already specified the schema with `search_path = bdjuno` on connect

Reply:
please also try to write this query with orm instead of raw sql